clcache.py

```diff
@@ -792,9 +792,9 @@ def _parseOptionsAndFiles(cmdline):
             'Ob', 'Gs', 'Fa', 'Fd', 'Fm',
             'Fp', 'FR', 'doc', 'FA', 'Fe',
             'Fo', 'Fr', 'AI', 'FI', 'FU',
-            'D', 'U', 'I', 'Zp', 'vm',
+            'D', 'U', 'I', 'Zp',
             'MP', 'Tc', 'V', 'wd', 'wo',
-            'W', 'Yc', 'Yl', 'Tp', 'we',
+            'Yc', 'Yl', 'Tp', 'we',
```

Comment:
Is `/W 4` really valid? I don't see any source for that. All documentation is talking about `/W4`. Is the space version used in the wild?